server/services/offender/index.js

```diff
@@ -120,9 +117,8 @@ const createOffenderService = (
   async function getVisitsRemaining({ prisonerId }) {
     try {
       logger.info(`OffenderService (getVisitsRemaining) - User: ${prisonerId}`);
-      const { remainingPvo = 0, remainingVo = 0 } =
-        await repository.getVisitBalances(prisonerId);
-      return { visitsRemaining: remainingPvo + remainingVo };
+      const aaa = await repository.getVisitBalances(prisonerId);
+      return { visitsRemaining: aaa.remainingPvo + aaa.remainingVo };
```

Comment:
Not the most descriptive name. Could de-structure to remove the need for a name I guess?
```
const { remainingPvo, remainingVo } = await repository.getVisitBalances(prisonerId);
return { visitsRemaining: remainingPvo + remainingVo };
```